src/components/AccountList.tsx

```diff
@@ -37,18 +39,26 @@ const AccountList: React.FC = () => {
       <Header>Accounts</Header>
       <Items>
         {project.accounts.map((account: Account, i: number) => {
+          const isActive = active == i
+          const contractName = getDeployedContracts(account) || "--"
+          const title = contractName !== "--"
+            ? `${contractName} is deployed to this account`
+            : `This account don't have any contracts`
           return (
             <Item
               key={i}
-              active={active === i}
+              title={title}
+              active={isActive}
               onClick={() => setActive(EntityType.Account, i)}
             >
               <AccountCard>
                 <Avatar seed={project.seed} index={i} />
                 <Stack>
                   <strong>0x{account.address.slice(-2)}</strong>
-                  <small>{getDeployedContracts(account) || "--"}</small>
+                  <small>{contractName}</small>
```

Comment:
```suggestion
                  <small>{contractName || "--"}</small>
```

Reply:
This computation is slightly above, where we defined `contractName` constant, so we don't need to make it "inline" 😅

Reply:
Nvm, I see what you suggest  👍